### PR TITLE
OCPBUGS-29223: Add DTK support for ARM kernel with 64k pages

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s): 4.16 only

Issue: https://issues.redhat.com/browse/OCPBUGS-29223

Link to docs preview: https://78057--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html

QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
@mike-nguyen PTAL

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -2462,6 +2462,8 @@ To retain the catalog rebuilding functionality, use `--rebuild-catalog`. However
 
 * Previously, during {product-title} updates in performance-tuned clusters, resuming a `MachineConfigPool` resource resulted in additional restarts for nodes in the pool. With this release, the controller reconciles against the latest planned machine configurations before the pool resumes, which prevents additional node reboots. (link:https://issues.redhat.com/browse/OCPBUGS-31271[*OCPBUGS-31271*])
 
+* Previously, ARM installations using Driver Tool Kit (DTK) provided the use of 4k pages in the kernel. With this release, DTK will also allow the installation of ARM kernels with 64k page size, providing a performance boost on the NVIDIA CPU. (link:https://issues.redhat.com/browse/OCPBUGS-29223[*OCPBUGS-29223*])
```
Comment (mike-nguyen) [inline release_notes/ocp-4-16-release-notes.adoc:2465]: Maybe something more like

> Previously, ARM installations used 4k pages in the kernel. With this release, support was added for installing 64k pages in the kernel at installation time only, providing a performance boost on the NVIDIA CPU. Driver Tool Kit (DTK)  was also updated to compile kernel modules for the 64k page size ARM kernel.

**Comment by mike-nguyen** [comment]
LGTM  (Note: this is not my normal area of QE)

**Review by sheriff-rh — APPROVED** [review]
Changes LGTM!

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**